src/main/java/uk/gov/hmcts/reform/professionalapi/domain/ProfessionalUser.java

```diff
@@ -6,17 +6,7 @@
 import java.util.List;
 import java.util.UUID;
 
-import javax.persistence.CascadeType;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.EntityListeners;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.ManyToOne;
-import javax.persistence.OneToMany;
-import javax.persistence.Transient;
+import javax.persistence.*;
```

Comment:
do no add .* in imports.

Reply:
this automatically happens

Reply:
you need to remove and specifically add the imports required..

Reply:
If you look at the latest files by choosing changes from all commits, this file has not been modified and this change is not in the PR